plugins/by-name/nvim-ufo/default.nix

```diff
@@ -84,6 +84,8 @@ lib.nixvim.plugins.mkNeovimPlugin {
 
       mappings = lib.nixvim.mkNullOrOption lib.types.attrs "Mappings for preview window.";
     };
+
+    setupLspCapabilities = lib.options.mkEnableOption "Setup LSP capabilities for nvim-ufo";
```

Comment:
This should be moved from `settingsOptions` to `extraOptions`.

`settings` is exclusively for the _upstream plugin's_ options. Out custom nixvim-specific options should live outside of `settings`.

Reply:
I'd also prefer it to be a `default=true` option.

`mkEnableOption "Setup LSP capabilities for nvim-ufo"` will produce:

```nix
mkOption {
  type = lib.types.bool;
  description = "Whether to enable Setup LSP capabilities for nvim-ufo.";
  default = false;
  example = true;
}
```

I'd suggest instead having:
```nix
mkOption {
  type = lib.types.bool;
  description = "Whether to enable setting-up LSP capabilities for nvim-ufo.";
  default = true;
  example = false;
}
```

Reply:
I've added the function `mkEnabledOption` to `lib/options.nix` since I'm planning to fix another plugin with a similar issue